admin-dev/themes/default/template/controllers/translations/helpers/view/translation_modules.tpl

```diff
@@ -56,12 +56,45 @@
 			<p>{l s='Expressions to translate:'} <span class="badge">{l s='%d' sprintf=[$count]}</span></p>
 			<p>{l s='Total missing expressions:'} <span class="badge">{l s='%d' sprintf=[$missing_translations]}</p>
 		</div>
+		<form action="{$url_submit_installed_module|escape:'html':'UTF-8'}" method="post" enctype="multipart/form-data" class="form-horizontal">
+			<div class="panel">
+				<input type="hidden" name="langue" value="{$lang}" />
+				<input type="hidden" name="type" value="{$type}" />
+				<input type="hidden" name="theme" value="{$theme}" />
+				<input type="hidden" name="controller" value="AdminTranslations" />
+				<h3>
+					<i class="icon-file-text"></i>
+					{l s='Modify translations'}
+				</h3>
+				<p class="alert alert-info">
+					{l s='Here you can modify translations for all installed module.'}<br />
+				</p>
+				<div class="form-group">
+					<label class="control-label col-lg-3" for="translations-languages">{l s='Select your module'}</label>
+					<div class="col-lg-4">
+						<select name="module" id="installed_module">
+							<option value="">{l s='Module'}</option>
+							{foreach from=$installed_modules key=key item=module}
```

Comment:
Yes this is the same, I wanted to use the key and I forget to remove it when I didn't use it.